nixos-modules/microvm/options.nix

```
@@ -466,6 +466,14 @@ in
       '';
     };
 
+    qemu.oomDeflate = mkOption {
```

[It looks like this feature is supported on more VMMs than just qemu.](https://github.com/firecracker-microvm/firecracker/blob/main/docs/ballooning.md) Let's make it generic:

```suggestion
    oomDeflate = mkOption {
```

Also, it has the same name `deflate_on_oom` so I wonder if `deflateOnOOM` is a more meaningful name for the option. YMMV.

---

Changed to deflateOnOOM and dropped the `qemu`